api-reference/10 UI Components/dxScheduler/1 Configuration/selectedCellData.md

```diff
@@ -21,6 +21,15 @@ This array contains objects with the following structure:
         }     
     }
 
```

Comment:
I think we should update line 8 above to avoid the plural possesive noun. Can we update it as follows?

Line 8:
The data of the currently selected cells.

Reply:
Noted